apps/fz_http/lib/fz_http/api_tokens.ex

```diff
@@ -0,0 +1,53 @@
+defmodule FzHttp.ApiTokens do
+  @moduledoc """
+  The ApiTokens context.
+  """
+
+  import Ecto.Query, warn: false
+  alias FzHttp.Repo
+
+  alias FzHttp.ApiTokens.ApiToken
+
+  def count, do: count(ApiToken)
+  def count(nil), do: nil
+
+  def count(queryable) when is_struct(queryable) or is_atom(queryable),
+    do: Repo.aggregate(queryable, :count)
+
+  def count(user_id), do: count(from(a in ApiToken, where: a.user_id == ^user_id))
+
+  def list_api_tokens do
+    Repo.all(ApiToken)
+  end
+
+  def list_api_tokens(user_id) do
+    Repo.all(from a in ApiToken, where: a.user_id == ^user_id)
+  end
+
+  def get_api_token(id), do: Repo.get(ApiToken, id)
+
+  def get_api_token!(id), do: Repo.get!(ApiToken, id)
+
+  def new_api_token(attrs \\ %{}) do
+    ApiToken.changeset(%ApiToken{}, attrs)
+  end
+
+  def create_api_token(attrs \\ %{}) do
+    FzHttp.Telemetry.create_api_token()
+    user_id = attrs[:user_id] || attrs["user_id"]
+
+    %ApiToken{}
+    |> ApiToken.changeset(attrs, count_per_user: count(user_id))
+    |> Repo.insert()
+  end
+
+  def expired?(%ApiToken{} = api_token) do
```

Comment:
We should choose the convention: do we add entity name to function names or not, right now we have functions like `create_api_token` and `expired?` but they should be either just `create` and `expired?` or `create_api_token` or `api_token_expired?`. (I would go with the first once where context is only responsible for one schema, 2nd one where context has multiple schemas.)
```suggestion
  def expired?(%ApiToken{} = api_token) do
```

Reply:
My Rails is showing. Makes sense!

Reply:
I'll stick to the second one in this PR since it's already quite large, and opened this in the backlog to get these refactored.